sdk/eventhub/Azure.Messaging.EventHubs/design/proposal-event-hubs-second-preview.md

```diff
@@ -1,389 +1,389 @@
-
-```csharp
-var client = CreateClient();
-var consumer = client.CreateConsumer(EventHubConsumer.DefaultConsumerGroupName, "fcbac12-43cda", EventPosition.Earliest);
-
-await foreach(var eventData in consumer.SubscribeToPartition(someCancellationToken))
-{
-    ProcessEvent(eventData);
-}
-```
-
-#### Subscribe to events as they are available, using a maximum wait time to return control
-
-```csharp
-var client = CreateClient();
-var consumer = client.CreateConsumer(EventHubConsumer.DefaultConsumerGroupName, "fcbac12-43cda", EventPosition.Earliest);
-var maximumWaitTime = TimeSpan.FromSeconds(2);
-
-await foreach(var eventData in consumer.SubscribeToPartition(maximumWaitTime, someCancellationToken))
-{
-    // If the maximum wait time elapsed before an event was 
-    // available, it will be null.  This returns control and allows
-    // for breaking out of the loop.
-    
-    if (eventData != null)
-    {
-        ProcessEvent(eventData);
-    }
-}
-```
-
-### Packages
-
-#### `Azure.Messaging.EventHubs`
-
-The main client library package, containing the core components for interacting with the Azure Event Hubs service.
-
-### Namespaces
-
-#### `Azure.Messaging.EventHubs`
-
-The top-level container for the types in the client library intended to be used by Event Hubs client library users. It is intended that types most frequently used by users for basic operations belong to this namespace to ensure ease of discovery.
-
-##### _Example types:_
- 
-- `RetryOptions`
-- `EventHubsRetryPolicy`
-- `BatchOptions`
-- `EventDataBatch`
-
-#### `Azure.Messaging.EventHubs.Errors`
-
-The location for exceptions and error-related information and operations. Many of these types are surfaced as information for consumers in response to conditions encountered during a basic operation. It is not intended that consumers need to create these types directly.
-
-##### _Example types:_
- 
-- `OperationCancelledException`
-- `TimeoutException`
-- `MessageSizeExceededException`
-- `QuotaExceededException`
-
-#### `Azure.Messaging.EventHubs.Core`
-
-The location for internal types used by the Event Hubs library to facilitate operations; these constructs are not intended to be consumed externally. 
-
-##### _Example types:_
- 
-- `DefaultEventHubsRetryPolicy`
-- `SystemMessagePropertyName`
-
-#### `Azure.Messaging.EventHubs.Amqp`
-
-The location for internal types used by the Event Hubs library to facilitate AMQP protocol-related activities; these constructs are not intended to be consumed externally. 
-
-##### _Example types:_
- 
-- `AmqpMessageConverter`
-- `AmqpEventDataBatch`
-
-#### `Azure.Messaging.EventHubs.Diagnostics`
-
-The location for internal types used by the Event Hubs library for diagnostics and logging activities; these constructs are not intended to be consumed externally. 
-
-##### _Example types:_
- 
-- `EventHubsEventSource`
-- `EventHubsDiagnosticSource`
+﻿# .NET Event Hubs Client: Track Two Proposal (Second Preview)
```

Comment:
Not sure what is up with this one;  the removal of the BOM prefix may have been preserved between branches.    /shrug